cram_pr2/cram_pr2_pick_place_demo/cram-pr2-pick-place-demo.asd

```diff
@@ -67,7 +67,9 @@
 
                cram-urdf-projection      ; for with-simulated-robot
                cram-urdf-projection-reasoning ; to set projection reasoning to T
+
```

Comment:
Should rearrange this to put cram-pr2-description at the bottom and group cram-urdf-xx together